book/src/themes.md

```diff
@@ -300,7 +300,7 @@ These scopes are used for theming the editor interface:
 | `ui.popup.info`                   | Prompt for multiple key options                                                                |
 | `ui.window`                       | Borderlines separating splits                                                                  |
 | `ui.help`                         | Description box for commands                                                                   |
-| `ui.text`                         | Command prompts, popup text, etc.                                                              |
+| `ui.text`                         | Default text style, command prompts, popup text, etc.                                          |
```

Comment:
Is it the default text for any text type? even editor text? or just the UI?
if it's only the UI, I think it might be best to use `UI text style` instead of `Default ...`

Reply:
Also for the editor text